docs/pages/releases/2021-11-01.mdx

```diff
@@ -0,0 +1,112 @@
+
+This surfaced a number of issues as **Apollo Sandbox** is hosted remotely rather than locally, including CORS issues and security concerns, making it a bad default.
+
+With this in mind, Keystone will now go back to using the **GraphQL Playground** by default instead of **Apollo Sandbox** as it did prior to updating to Apollo Server 3.
+
+We have now introduced the `graphql.playground` config option, with three possible settings:
+
+* `true` will configure Apollo Server plugins to enable the GraphQL Playground
+* `false` will configure Apollo Server plugins to disable any `GET` requests to the GraphQL endpoint
+* `'apollo'` will add no plugins to Apollo Server, enabling the new Apollo Sandbox behaviour
+
+`graphql.playground` defaults to `process.env.NODE_ENV !== 'production'` which matches the previous Keystone 6 behaviour before the October update and Apollo Server 3.
+
+## CORS Configuration <Emoji symbol="🌐" alt="World" />
+
+We've removed the `graphql.cors` option, we had one too many ways to configure CORS and it was proving to be confusing.
+
+You should now exclusively configure `cors` with the `server.cors` option.
+
+## Relationship Filtering <Emoji symbol="❤️" alt="Heart" />
+
+If you have hundreds of items in your relationship fields, the Admin UI was sometimes displaying duplicate entries and/or missing entries.
+
+We've made a series of improvements to fetching data resulting in a performance boost, as well as filtering fixes. We also changed the way we detect when an ID is pasted into the field allowing you to select a related item quickly.
+
+## Removed `resolveFields` <Emoji symbol="🚧" alt="Construction" />
+
+The deprecated `resolveFields` from `context.query` has been removed.
+
+If you were still using it, you should switch to providing `the query option` to `context.query` or use `context.db` if you were providing `false`.
+
+The `context.query` functions will now also throw an error if an empty string is passed to `query` rather than silently returning what the `context.db` functions return, you must select at least one field or omit the `query` option to default to selecting the `id`.
+
+## Internal types renamed <Emoji symbol="✏️" alt="Pencil" />
+
+We have updated `@graphql-ts/schema` to `0.5.0`.
+
+The `__rootVal` properties on `ObjectType`, `InterfaceType` and `UnionType` have been renamed to `__source`, this is intended to be internal but you may have depended on it, if you were using it, rename `__rootVal` to `__source`.
+
+In addition the `fields` property on `InterfaceType` has been renamed to `__fields` and it will no longer exist at runtime like the other types.
+
+## GraphQL Schema Reorder <Emoji symbol="🛗" alt="Elevator" />
+
+The way the `endSession` field on the `Mutation` type and the `keystone` field on the `Query` type have been adjusted. This may result in re-ordering in your generated `schema.graphql` file. The `sessionSchema` export of `@keystone-next/keystone/session` has also been removed.
```

Comment:
"The way the `endSession` field on the `Mutation` type and the `keystone` field on the `Query` type have been adjusted."

^ I find this hard to understand. 

Is it the `order` of those things?

Reply:
I tweaked this a little.